lib/core/http.js

```diff
@@ -74,5 +74,31 @@ module.exports = (config) => {
     );
   };
 
-  return { getCageKey, runCage, getCert };
+  const postMetric = async (count) => {
+    const metricsUrl = `metrics/sdkEncryptions?sdk=node&numEncryptions=${count}`;
```

Comment:
Do we want a more generic mechanism for reporting client metrics? Crafting urls for each value will become problematic at scale. Perhaps we should be posting batches of values to a single url?

Reply:
I think it should be fine for now, being more work than helpful

Reply:
At the very least I would pass the metric name as an arg to the function